xmtp_api_d14n/Cargo.toml

```diff
@@ -26,33 +26,25 @@ xmtp_common.workspace = true
 xmtp_configuration = { workspace = true }
 xmtp_proto = { workspace = true, features = ["convert"] }
 
-# only for tests
-xmtp_api_http = { workspace = true, optional = true, features = ["test-utils"] }
+# Test Utils
+xmtp_api_grpc = { path = "../xmtp_api_grpc", optional = true, default-features = false }
```

Comment:
this and all other occurrences of using local deps should use `workspace = true` instead of specifying the path.